xml/System.IO/Directory.xml

```diff
@@ -4006,27 +4006,29 @@ An I/O error occurred.</exception>
       </Parameters>
       <Docs>
         <param name="sourceDirName">The path of the file or directory to move.</param>
-        <param name="destDirName">The path to the new location for <paramref name="sourceDirName" />. If <paramref name="sourceDirName" /> is a file, then <paramref name="destDirName" /> must also be a file name.</param>
+        <param name="destDirName">The path to the new location for <paramref name="sourceDirName" /> or its contents. If <paramref name="sourceDirName" /> is a file, then <paramref name="destDirName" /> must also be a file name.</param>
         <summary>Moves a file or a directory and its contents to a new location.</summary>
         <remarks>
           <format type="text/markdown"><![CDATA[  
   
-## Remarks  
- This method creates a new directory with the name specified by `destDirName` and moves the contents of `sourceDirName` to the newly created destination directory. If you try to move a directory to a directory that already exists, an <xref:System.IO.IOException> will occur. For example, an exception will occur if you try to move c:\mydir to c:\public, and c:\public already exists. Alternatively, you could specify "c:\\\public\\\mydir" as the `destDirName` parameter, provided that "mydir" does not exist under "c:\\\public", or specify a new directory name such as "c:\\\newdir".  
+## Remarks
+
+This method creates a new directory with the name specified by `destDirName` and moves the contents of `sourceDirName`, including files and directories, to the newly created destination directory. It then deletes the `sourceDirName` directory.
```

Comment:
I am not a native speaker, but to me the best word that describes what this method does is "rename".

```suggestion
The method renames the directory using provided paths. It creates a new directory with the name specified by `destDirName` and moves the contents of `sourceDirName`, including files and directories, to the newly created destination directory. It then deletes the `sourceDirName` directory.
```

@gewarren what do you think?

Reply:
@adamsitnik I don't see it as a straight rename. For example, you could move c:\mydir to c:\sub1\newfolder, in which case it's moved *and* renamed.